mlir/test/Dialect/MIOpen/lowering_memref_kcyx_nchw_nkhw.mlir

```diff
@@ -44,6 +44,11 @@ func @miopen_conv2d_bwd_data_kcyx_nchw_nkhw(%filter : memref<1x128x8x3x3xf32>, %
 // CHECK-NEXT:  {{miopen.transform.*{.*}.*memref.*memref}}
 // CHECK-NEXT:  {{miopen.transform.*{.*}.*memref.*memref}}
 // CHECK-NEXT:  {{miopen.transform.*{.*}.*memref.*memref}}
+// CHECK-NEXT:  {{miopen.transform.*{.*}.*memref.*memref}}
```

Comment:
please add operation names check for these newly added `miopen.transform` ops.

Reply:
same as above. please help improve checks here, use regex to check the attributes applied to `miopen.transform`.